### PR TITLE
ENG-12957: Add section for S3 File Browser configuration

### PR DESCRIPTION
**Pull request by VictorGFM**
Add a section for S3 File Browser configuration, instructing how to configure the corresponding sidecar template parameters to work with S3 File Browser.

### CODE REVIEW

**Review by wcmjunior — CHANGES_REQUESTED** [review]
Same observation is the other PRs: this new section should live on a separate `docs` page referred from the `Advanced` section of the main README.

File: README.md [hunk 1]
```diff
@@ -174,6 +174,29 @@ as part of the [repository configuration](https://cyral.com/docs/manage-reposito
 
 See the next section for more details about the values file parameters.
 
+#### Enable the S3 File Browser
+
+To configure the sidecar to work on the S3 File Browser, set the following extra parameters under the service key of your values file:
+
+```
+service:
+  dnsName: "<CNAME>" # ex: "www.sidecar-custom-name.com"
```
[the remark this answers is not in the document]
Comment (VictorGFM) [reply README.md]: Addressed: [Move s3 documentation to /docs directory](https://github.com/cyral-quickstart/quickstart-sidecar-helm/pull/7/commits/e3ec5608d65fde017d3456b118e7c4d3a56e1f71)

File: README.md [hunk 2]
```diff
@@ -237,3 +214,4 @@ Instructions for advanced configurations are available for the following topics:
 * [Sidecar instance metrics](./docs/metrics.md)
 * [Use a pre-existing service account](./docs/pre-existing-sa.md)
 * [Values file reference](./docs/values-file.md)
+* [Enable the S3 File Browser](./docs/s3-browser.md)
```
Comment (wcmjunior) [inline README.md]: Please keep the items in alphabetical order.

File: docs/s3-browser.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  dnsName: "<CNAME>" # ex: "sidecar.custom-domain.com"
+  loadBalancer:
+    tlsPorts: [443]
+    certificateId: "arn:aws:acm:<REGION>:<AWS_ACCOUNT>:certificate/<CERTIFICATE_ID>"
+```
+
+For the helm template, there is no equivalent to the terraform’s
+`sidecar_dns_hosted_zone_id` variable. It means the CNAME will need to
+be created after the deployment. See [Add a CNAME or A record for the
+sidecar](https://cyral.com/docs/sidecars/manage/alias).
```
Comment (wcmjunior) [inline docs/s3-browser.md:16]:
```suggestion
The CNAME provided in `service.dnsName` must be created
after the deployment pointing to the sidecar load balancer.
See [Add a CNAME or A record for the sidecar](https://cyral.com/docs/sidecars/manage/alias).
```

**Review by wcmjunior — APPROVED** [review]

**Pull request merged**